docs/getting_started/RuntimeWorkflow.md

```diff
@@ -0,0 +1,85 @@
+## Runtime Workflow
+## Prerequisites
+   Cluster(kind or EKS) should be deployed and available to the nexus runtime deployment
+
+* [Nexus Install](RuntimeWorkflow.md#nexus-install)
+
+* [Nexus Pre-req verify](RuntimeWorkflow.md#nexus-pre-req-verify)
+
+* [Setup Environment](RuntimeWorkflow.md#setup-environment)
+
+* [Install Nexus Admin Runtime](RuntimeWorkflow.md#install-nexus-admin-runtime)
+  
+* [Install Nexus Tenant Runtime](RuntimeWorkflow.md#install-nexus-tenant-runtime)
+
+## Nexus Install
+
+Install Nexus CLI
+
+```
+curl -LJ https://github.com/vmware-tanzu/graph-framework-for-microservices/releases/latest/download/nexus-$(uname -s | awk '{print tolower($0)}')_$(go env GOARCH)  -o nexus && chmod +x nexus
+```
+
+## Nexus Pre-req Verify
+
+Verify nexus sdk pre-requisites are satisfied
+
+    nexus prereq verify
+
+<!-- nexus-specific exports
+```
+# store the current directory before we `cd` into the app dir
+export DOCS_INTERNAL_DIR=$PWD/docs/_internal
+```
+-->
+
+## Install Nexus Admin Runtime
+
+:bulb: This step is optional, based on the user's requirement, this step shall be skipped or executed
```

Comment:
what are the requirements? this can be confiusing to user if he should do this step or not